src/components/accordion.scss

```diff
@@ -10,6 +10,12 @@
     gap: var(--op-space-2x-small);
     grid-template-columns: auto 1fr auto;
 
+    &::marker,
+    &::-webkit-details-marker {
```

Comment:
@Jeremy-Walton It's such a shame that this needs to be done for safari but at least it will work now and once safari supports the standard it will fall back to that and we can remove this at that time. Thanks for digging into this and finding the solution.